.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Ignore bulk black/ruff formatting fixes
+4fd97fde45b15bb8f49a7acebcdf9d4ef5afd0f4
```

Comment:
This commit ID looks like it may need updating - the commit where the formatting changes are made is now different

Reply:
oop